assets/app/scripts/services/data.js

```diff
@@ -1019,8 +1066,15 @@ angular.module('openshiftConsole')
       namespace: namespace,
       q: params
     };
+    // TODO (bpeterse): The URL_OBJECT_SUBRESOURCE & URL_NAMESPACED_OBJECT_SUBRESOURCE
+    // have both been removed.  Need to figure out what to use now in place of these.
```

Comment:
Ah, yes, glad I left that there.  @jwforres you prob can confirm this. I updated the constants here based on comparison to usage elsewhere... it is in reference to Sam's next comment below.

```javascript
// old
template = namespaceInPath ? URL_NAMESPACED_WATCH_LIST : URL_WATCH_LIST;
// new
template = namespaceInPath ? URL_NAMESPACED_OBJECT : URL_OBJECT;
```